misk/web/tabs/web-actions/src/rewrite/WebActionCardBody.tsx

```diff
@@ -3,8 +3,9 @@ import { Menu, H5, UL } from "@blueprintjs/core"
 import { WebActionMetadata } from "./types"
 import WebActionCollapse from "./WebActionCollapse"
 import WebActionSendRequest from "./WebActionSendRequest"
-import WebActionParameters from "./WebActionParameters";
-import WebActionDescription from "./WebActionDescription";
+import WebActionRequestParameters from "./WebActionRequestParameters"
+import WebActionDescription from "./WebActionDescription"
+import WebActionResponseType from "./WebActionResponseType";
```

Comment:
nit: rm semicolon

Reply:
I'm finding that IntelliJ doesn't format the code in the way I'd expect when automatically importing. I probably need to set some options here. Do you have any tips?

Reply:
I think I've got this sorted now. Thanks for the pickup.